cli/tests/extensions.rs

```diff
@@ -0,0 +1,317 @@
+#![cfg(feature = "extensions")]
+
+use std::convert::TryFrom;
+use std::env;
+use std::ffi::OsStr;
+use std::fs;
+use std::path::{Path, PathBuf};
+
+use assert_cmd::Command;
+use phylum_cli::commands::extensions::*;
+use rand::Rng;
+use regex::Regex;
+
+////////////////////////////////////////////////////////////////////////////////
+// Acceptance criteria tests
+////////////////////////////////////////////////////////////////////////////////
+
+// When a user runs `phylum extension add .`, the extension in the current
+// working directory should be installed.
+#[test]
+fn extension_is_installed_correctly() {
+    let tmp_dir = TmpDir::new();
+    Command::cargo_bin("phylum")
+        .unwrap()
+        .env("XDG_DATA_HOME", &tmp_dir)
+        .arg("extension")
+        .arg("add")
+        .arg(fixtures_path().join("sample-extension"))
+        .assert()
+        .success();
+
+    env::set_var("XDG_DATA_HOME", &tmp_dir);
```

Comment:
Does setting this variable mean we can't run the tests in parallel? Can we accomplish what we need by running external commands?